Source/Logger.swift

```diff
@@ -0,0 +1,68 @@
+import Venice
+public class Logger {
+    
+    //    #namespace     String   The name of the namespace / module
+    //    #file          String   The name of the file in which it appears.
+    //    #line          Int      The line number on which it appears.
+    //    #column        Int      The column number in which it begins.
+    //    #function      String   The name of the declaration in which it appears.
+    //    #dsohandle     String   The dso handle.
+    
+    var appenders = [Appender]()
+    var levels : Log.Level
+    var name : String
+    
+    public init(name: String, appender: Appender, levels: Log.Level = .all) {
+        appenders.append(appender)
+        self.levels = levels
+        self.name = name
+    }
+    
+    public init(name: String, appenders: [Appender], levels: Log.Level = .all) {
+        self.appenders.append(contentsOf: appenders)
```

Comment:
Instead of having an overload for a single appender, its probably a better idea to use `appenders: Appender...` and append that instead. Also, I believe it is more "swifty" to use `self.appenders += appenders` instead of `.append`, but that's not a big deal.